app/controllers/articles_controller.rb

```diff
@@ -9,6 +9,9 @@ def index
     else
       @articles = Article.all
     end
+    if params[:sorted]
```

Comment:
sorted is not a great param. I'd do order_by with different options, Right now it's sorta hard to figure out what you're sorting by. Also, consider using order, cause right now afaik, sort_by sorts in Ruby, while order should sort in SQL, which is far far more efficient

Reply:
so like order_by=date, order_by=title, etc.

Reply:
https://stackoverflow.com/questions/14897239/are-the-order-method-parameters-in-activerecord-sanitized-by-default

Reply:
Only sort by rank and date for now.